### PR TITLE
i18n(es): Translation of `buttercms.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `buttercms.mdx`to spanish.

### CODE REVIEW

File: src/content/docs/es/guides/cms/buttercms.mdx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+---
+<body>
+	{items.map(item => <div>
+		<h2>{item.name} - ${item.price}</h2>
+		<p set:html={item.description}></p>
+	</div>)}
+</body>
+```
+
+La interfaz refleja los tipos de campo. El campo WYSIWYG `description` se carga como una cadena de HTML y [`set:html`](/en/reference/directives-reference/#sethtml) te permite renderizarlo.
```
Comment (Yan-Thomas) [inline src/content/docs/es/guides/cms/buttercms.mdx:96]:
```suggestion
La interfaz refleja los tipos de campo. El campo WYSIWYG `description` se carga como una cadena de HTML y [`set:html`](/es/reference/directives-reference/#sethtml) te permite renderizarlo.
```

**Comment by dreyfus92** [comment]
thanks @Yan-Thomas for fixing the broken link.

File: src/content/docs/es/guides/cms/buttercms.mdx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+<html>
+  <title>{fields.seo_title}</title>
+  <body>
+    <h1>{fields.headline}</h1>
+    <img src={fields.hero_image} />
+  </body>
+</html>
+```
+
+## Official Resources
```
Comment (raul-repos) [inline src/content/docs/es/guides/cms/buttercms.mdx:123]:
```suggestion
## Recursos Oficiales
```

**Review by raul-repos — COMMENTED** [review]
Muy buen trabajo!!

**Comment by dreyfus92** [comment]
Muchas gracias @raul-repos por el review 😁

**Review by Yan-Thomas — APPROVED** [review]
Thanks for this @dreyfus92 & @raul-repos, LGTM!

**Pull request merged**